content/02.intro.md

```diff
@@ -1,11 +1,11 @@
 ## Introduction {#intro}
 
-Deep learning (DL), a subfield of machine learning (ML) implementing artificial neural networks with many layers, is increasingly used for the analysis of biological data [@doi:10.1098/rsif.2017.0387].
+Deep learning (DL), a subfield of machine learning (ML) focusing on the implementation of artificial neural networks with many layers, which are increasingly used for the analysis of biological data [@doi:10.1098/rsif.2017.0387].
```

Comment:
How about just "focusing on artificial neural networks"? I actually think "implementing" unnecessarily places DL in the realm of "software", but that needn't be the case.

Reply:
I strongly agree. Prob. got biased by reading "implemented" in the original draft, but it is really unnecessary like you suggest.